packages/react/src/Tabs/TabsContext.tsx

```diff
@@ -0,0 +1,16 @@
+import { createContext } from 'react'
+
+export type TabsContextValue = {
+  activeTab: number
+  // eslint-disable-next-line no-unused-vars
+  updateTab: (tab: number) => void
+}
+
+const defaultValues: TabsContextValue = {
+  activeTab: 0,
+  updateTab: () => {},
+}
+
+const TabsContext = createContext(defaultValues)
+
+export default TabsContext
```

Comment:
We use named exports, or is there a reason to deviate here? Same for `AccordionContext` and `useFocusWithArrows`.